packages/safe-apps-web3-react/README.md

```diff
@@ -0,0 +1,41 @@
+# Safe Apps Web3-react connector
+
+A connector to be used with web3-react package
+
+## Installation
+
+```bash
+yarn add @gnosis.pm/safe-apps-web3-react
+
+npm install @gnosis.pm/safe-apps-web3-react
+```
+
+## Usage
+
+```js
+import { SafeAppConnector } from '@gnosis.pm/safe-apps-web3-react';
+```
+
+The connector follows web3-react's connectors API convention. Visit web3-react [repo](https://github.com/NoahZinsmeister/web3-react) for more details
+
+### Helper hook
+
+You can use our helper hook to automatically connect to a safe, it will automatically connect to the Safe if it detects that it's loaded in Safe App context:
+
+```js
+import { useSafeAppConnection } from '@gnosis.pm/safe-apps-web3-react';
+
+const App = () => {
+  const triedToConnectToSafe = useSafeAppConnection();
```

Comment:
Does this require a parameter (connector)?

https://github.com/gnosis/safe-apps-sdk/pull/122/files#diff-6fdbd8820b0ce753e3cd26c61685a0fd2e5324e813f1bd1e42fdee690f47dcfbR5